src/Tribe/Cost_Utils.php

```diff
@@ -321,7 +321,10 @@ public function parse_cost_range( $costs, $max_decimals = null, $sort = true ) {
 
 		foreach ( $costs as &$cost ) {
 			// Get the required parts
-			if ( preg_match_all( '/' . $price_regex . '/', $cost, $matches ) ) {
+			if (
+				is_numeric( $cost )
+				&& preg_match_all( '/' . $price_regex . '/', $cost, $matches )
```

Comment:
I'm not completely clear on why test here to see if `$cost` is numeric ... the `$price_regex` is designed to accommodate and work with strings such as `1,234,567.89` which would fail that test.

Reply:
Let me re-word ... if we only accept strings and numbers that pass the `is_numeric()` test, why parse with the regex?